easybuild/easyconfigs/t/Tk/Tk-8.6.4-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.4'
+
```

Comment:
I think the fact that the trickery in matplotlib's `setupext.py` doesn't work may be a sign of something else being broken in Tk...

there's a whole bunch of things that are tried by matplotlib, and they silently ignore some attempts that fail hard (with exceptions), but we should probably figure out what fails and why?